packages/contracts-bedrock/contracts/test/L1Block.t.sol

```diff
@@ -91,5 +105,141 @@ contract L1BlockTest is CommonTest {
             _l1FeeOverhead: type(uint256).max,
             _l1FeeScalar: type(uint256).max
         });
+
+        assertEq(lb.number(), type(uint64).max);
+        assertEq(lb.timestamp(), type(uint64).max);
+        assertEq(lb.basefee(), type(uint256).max);
+        assertEq(lb.hash(), keccak256(abi.encode(1)));
+        assertEq(lb.sequenceNumber(), type(uint64).max);
+        assertEq(lb.batcherHash(), bytes32(type(uint256).max));
+        assertEq(lb.l1FeeOverhead(), type(uint256).max);
+        assertEq(lb.l1FeeScalar(), type(uint256).max);
+        assertEq(lb.getL1BlockHash(type(uint64).max), keccak256(abi.encode(1)));
+    }
+
+    function test_getL1BlockHash_upperBound_l1BlockHashNumberOutOfBounds_reverts() external {
+        vm.expectRevert(L1BlockHashNumberOutOfBounds.selector);
+        lb.getL1BlockHash(2); // we set the block number to 1 in the
+        // setup so we are expecting a revert here
+    }
+
+    function test_getL1BlockHash_lowerBound_l1BlockHashNumberOutOfBounds_reverts() external {
```

Comment:
this can be changed to a fuzz test where you would set the l1blockvalues from 2 numbers provided.
The first one you would assume that it is uint256.max-BLOCK_HASHES_SIZE and the second one would be lower than the first one then query for `second_value` - expect that it will revert.